pokerole_irl/pokerole_app/migrations/0015_alter_pokedexentry_description.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.18 on 2023-05-01 16:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pokerole_app', '0014_pokedexentry_description_and_rarity_change'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='pokedexentry',
+            name='description',
+            field=models.TextField(blank=True, default=''),
+        ),
+    ]
```

Comment:
Where does this change come from? Nothing seems to have changed about this model in this PR

Reply:
no clue what happened. I assume that I must have done something when I was looking a the pokedex models. I'll ignore this when merging.

Reply:
It's probably better to delete this file altogether.